django/db/backends/oracle/introspection.py

```diff
@@ -156,7 +156,7 @@ def get_table_description(self, cursor, table_name):
         field_map = {
             column: (
                 display_size,
-                default if default != "NULL" else None,
+                default.rstrip() if default != "NULL" else None,
```

Comment:
This is extremely funny behavior, for non-string primitives Oracle returns everything after `DEFAULT` as a data default, e.g.
- `DEFAULT 345 NOT NULL` -> `"345 "`,
- `DEFAULT (345) NOT NULL` -> `"(345) "`,
- `DEFAULT (345)` -> `"(345)"`,
- `DEFAULT 345` -> `"345"`,
- `DEFAULT "string" NOT NULL` -> `"'string'"`.

Introspection should also be fixed in other cases, but we can leave for follow up PRs.